docs/modules/ROOT/pages/config-file.adoc

```diff
@@ -12,6 +12,8 @@ include::partial$mrdocs-example.yml[]
 
 The xref:usage.adoc[Usage] page provides a detailed explanation of what to combine options from the configuration file and the command line.
 The <<config-options-reference>> section provides a detailed explanation of the options available.
+To get linting and autocompletion in the config file, the https://github.com/redhat-developer/yaml-language-server[YAML language server] can be used.
```

Comment:
This deserves a section including where the file can be found, the specification, etc... 

If I understand correctly, as the file follows that json spec, many IDEs support this format even without the user having to know about anything.

The most important/actionable information is that the user needs to put that comment at the top of the file.